### ISSUE TITLE
std out of range error

### ISSUE THREAD

**Issue by Twilight-Shuxin** [uwsaml/nexus#24]
Hi, I come across this error here today. I pulled the docker image and tried to profile the model, but it gave me this error. I wonder what's the problem? I am running the model on an A100 gpu, is it not supported by Nexus? Thanks so much if anyone could help me with this issue! 

<img width="864" alt="image" src="https://github.com/uwsampl/nexus/assets/34962039/9a66f0f6-3c13-4a42-964d-c06c860f57f4">

**Comment by yukiki-jc** [comment]
Hi, @Twilight-Shuxin  I am also experiencing this problem. Have you sovled this yet?

**Comment by Twilight-Shuxin** [comment]
> Hi, @Twilight-Shuxin I am also experiencing this problem. Have you sovled this yet?

No, I gave up after that. I guess it is the architecture change on A100 which led to this failure. If you have chance to test it on a V100, it may show whether this guess is correct. 💔
